services/core-web/common/reducers/mineReducer.ts

```diff
@@ -15,7 +15,7 @@ interface MineState {
   mineGuid: string;
   mineBasicInfoList: IMine[];
   mineDocuments: IMineDocument[];
-  subscribedMines: IMine[];
+  subscribedMines: { records: IMine[]; loaded: boolean };
```

Comment:
The redux state logic doesn't update synchronously without tying this in, making my skeleton components "jump" as it loads. There's some room for discussion on how exactly to structure it though, basically when to separate it into 2 different functions or properties (search getGlobalMineAlerts to compare). Having the selector return an object with 2 properties (records & loaded) means we create fewer functions, but having 2 selectors like getXs and getXsLoaded is easier to implement retroactively without breaking anything.